app/components/concept-page/question-card.hbs

```diff
@@ -4,14 +4,19 @@
       {{markdown-to-html @question.queryMarkdown}}
     </div>
 
-    <div class="flex flex-col -mx-3 sm:-mx-4 gap-px" {{did-insert this.handleDidInsertOptionsList}}>
+    {{! template-lint-disable no-invalid-interactive }}
+    <div
+      class="question-card-option-container flex flex-col -mx-3 sm:-mx-4 gap-px"
+      {{did-insert this.handleDidInsertOptionsList}}
+      {{on "keydown" this.handleKeydown}}
```

Comment:
@libmartinito can't this be implemented using `{{on-key}}`? (Look at how the other keyboard shortcuts like Enter / Backspace are implemented)